packages/libraries/cli/src/commands/introspect.ts

```diff
@@ -42,10 +46,14 @@ export default class Introspect extends Command<typeof Introspect> {
       {} as Record<string, string>,
     );
 
-    const schema = await loadSchema(args.location, {
-      headers,
-      method: 'POST',
-    }).catch(err => {
+    const schema = await loadSchema(
+      flags['ignore-federation'] === true ? 'introspection' : 'federation-subgraph-introspection',
```

Comment:
Instead of needing to be configured, could we first try `_service` and if we get back a type error, then fallback to the old introspection query?
I think having this be seamless would provide a better UX.

Reply:
It tries to fetch _sdl and next it introspects by default. The flag that skips the _sdl is opt-in. It's a breaking change so we need to at least give an option to switch back to the previous behaviour, if needed.

Reply:
but I'm also fine with releasing a new major and remove the opt-in option. If you want to push this PR further, I more than happy to pass it to you